src/App.jsx

```diff
@@ -294,8 +294,15 @@ function App() {
           </div>
         )}
       </div>
-      <Dialog open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
-        <DialogTitle>New Upload</DialogTitle>
+      <Dialog style={{borderRadius: "100px"}} open={openNewUpload} onClose={() => setOpenNewUpload(false)}>
```

Comment:
using reserved prop-names like className and style(in this case) on custom components hurts readability and makes the code harder to maintain. These props should only be used for DOM nodes like div, a, span, etc.
You can take a look into this [mui.com/material-ui/customization/how-to-customize](https://mui.com/material-ui/customization/how-to-customize/) and fix it.